pkgs/development/tools/coder/update.sh

```diff
@@ -5,9 +5,9 @@ set -eu -o pipefail
 
 cd "$(dirname "${BASH_SOURCE[0]}")"
 
-# Fetch the latest stable version
-LATEST_STABLE_TAG=$(curl ${GITHUB_TOKEN:+" -u \":$GITHUB_TOKEN\""} --silent https://api.github.com/repos/coder/coder/releases/latest | jq -r '.tag_name')
-LATEST_STABLE_VERSION=$(echo ${LATEST_STABLE_TAG} | sed 's/^v//')
+# Fetch the latest stable version. We have to sort by the highest
+# version because we may release patches for older versions.
```

Comment:
```suggestion
```
The code is self-explanatory.

Where is the GITHUB_TOKEN? As the bot does far too many requests, it may fail silently. It is safer to include it.

Reply:
Oop, yes forgot to add it!

Reply:
Fixed!